kernel/libs/aster-bigtcp/src/socket/bound.rs

```diff
@@ -318,21 +322,8 @@ pub enum ConnectState {
     Refused,
 }
 
-#[derive(Debug, Clone, Copy)]
-pub struct NeedIfacePoll(bool);
-
-impl NeedIfacePoll {
-    pub const TRUE: Self = Self(true);
-    pub const FALSE: Self = Self(false);
-}
-
-impl Deref for NeedIfacePoll {
-    type Target = bool;
-
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}
+/// Indicates whether the iface needs to be polled
```

Comment:
A bit more consistency with the doc comments of `TcpConnBecameDead`, IMO:
```suggestion
/// Whether the iface needs to be polled.
```